editor/src/clj/editor/code/view.clj

```diff
@@ -2657,7 +2823,7 @@
                                              :font-size (.getValue font-size-property)
                                              :font-name  (.getValue font-name-property)
                                              :grammar grammar
-                                             :gutter-view (CodeEditorGutterView.)
+                                             :gutter-view (->CodeEditorGutterView)
```

Comment:
Using `(->CodeEditorGutterView)` instead of `(CodeEditorGutterView.)` allows us to skip re-evaluating the `make-view!` defn after we redefine `CodeEditorGutterView` class.